rotkehlchen/chain/ethereum/modules/yearn/accountant.py

```diff
@@ -2,46 +2,70 @@
 
 from rotkehlchen.accounting.structures.evm_event import get_tx_event_type_identifier
 from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
-from rotkehlchen.chain.evm.accounting.interfaces import ModuleAccountantInterface
-from rotkehlchen.chain.evm.accounting.structures import TxAccountingTreatment, TxEventSettings
+from rotkehlchen.chain.evm.accounting.interfaces import DepositableAccountantInterface
+from rotkehlchen.chain.evm.accounting.structures import TxEventSettings
 
 from .constants import CPT_YEARN_V1, CPT_YEARN_V2
 
 if TYPE_CHECKING:
     from rotkehlchen.accounting.pot import AccountingPot
 
 
-class YearnAccountant(ModuleAccountantInterface):
+class YearnAccountant(DepositableAccountantInterface):
 
     def event_settings(self, pot: 'AccountingPot') -> dict[str, 'TxEventSettings']:
         """Being defined at function call time is fine since this function is called only once"""
         return {
+            get_tx_event_type_identifier(HistoryEventType.RECEIVE, HistoryEventSubType.RECEIVE_WRAPPED, CPT_YEARN_V1): TxEventSettings(  # noqa: E501
+                taxable=False,
+                count_entire_amount_spend=False,
+                count_cost_basis_pnl=False,
+                method='acquisition',
+            ),
             get_tx_event_type_identifier(HistoryEventType.DEPOSIT, HistoryEventSubType.DEPOSIT_ASSET, CPT_YEARN_V1): TxEventSettings(  # noqa: E501
                 taxable=False,
                 count_entire_amount_spend=False,
                 count_cost_basis_pnl=False,
                 method='spend',
-                accounting_treatment=TxAccountingTreatment.SWAP,
+                accountant_cb=self.process_deposit,
             ),
-            get_tx_event_type_identifier(HistoryEventType.WITHDRAWAL, HistoryEventSubType.RETURN_WRAPPED, CPT_YEARN_V1): TxEventSettings(  # noqa: E501
+            get_tx_event_type_identifier(HistoryEventType.SPEND, HistoryEventSubType.RETURN_WRAPPED, CPT_YEARN_V1): TxEventSettings(  # noqa: E501
```

Comment:
It's part of the consistency change. We tend to use **SPEND/RETURN_WRAPPED** for LP token returned.
So since I changed the decoder, it was only normal to change the accountant settings.